src/Maker/MakeCrud.php

```diff
@@ -163,6 +163,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             Request::class,
             Response::class,
             Route::class,
+            EntityManagerInterface::class,
```

Comment:
Further above, there is a `$repositoryClassName`. In some cases, this is equal to `EntityManagerInterface`. So we should check if it is and, if so, avoid adding `EntityManagerInterface`. Or, hmm, perhaps better: we could improve `UseStatementGenerator` to ignore duplicates.

Reply:
Thank you for your, it was so simple and right in front of my eyes but I hadn't even seen...

I'll make a note to ignore duplications in UseStatementGenerator for another PR, or in this PR if I have time